config/activator.yaml

```diff
@@ -78,10 +78,10 @@ spec:
           # The numbers are based on performance test results from
           # https://github.com/knative/serving/issues/1625#issuecomment-511930023
           requests:
-            cpu: 300m
+            cpu: 800m
```

Comment:
This seems a bit high, I'd actually keep the starting value as is.

Reply:
How about the target value to trigger HPA scaling up?

Reply:
Changed back to 300m and increase targetAverageUtilization to 250.